service/src/test/java/greencity/ModelUtils.java

```diff
@@ -2253,6 +2253,7 @@ public static EditServiceDto getEditServiceDto() {
             .locationId(1L)
             .price(100)
             .description("test")
+            .descriptionEng("test")
```

Comment:
could you please use different values for `description` and `descriptionEng` to be sure everything works as expected?